### PR TITLE
spectrum picker and chevron icon added

### PR DESCRIPTION
**Pull request by draganatrajkovic**
From spectrum components added:
* sp-picker
* sp-icon-chevron100
* sp-icon-chevron-down

We use them for our new blocks.

**Test URLs:**
- Before: https://stage--milo--adobecom.hlx.live/?martech=off
- After: https://spectrum-picker-and-chevron-icon--milo--draganatrajkovic.hlx.page/?martech=off

Preview:
- Before:  https://partner-news-block--dx-partners--adobecom.hlx.page/SPP/drafts/dragana/partner-news?martech=off
- After:  https://partner-news-block--dx-partners--adobecom.hlx.page/SPP/drafts/dragana/partner-news?milolibs=spectrum-picker-and-chevron-icon--milo--draganatrajkovic&martech=off

### CODE REVIEW

File: libs/features/spectrum-web-components/src/icons-ui.js [hunk 1]
```diff
@@ -13,3 +13,4 @@ import '@spectrum-web-components/icons-ui/icons/sp-icon-cross200.js';
 import '@spectrum-web-components/icons-ui/icons/sp-icon-cross300.js';
 import '@spectrum-web-components/icons-ui/icons/sp-icon-cross400.js';
 import '@spectrum-web-components/icons-ui/icons/sp-icon-cross500.js';
+import '@spectrum-web-components/icons-ui/icons/sp-icon-chevron100.js';
```
Comment (yesil) [inline libs/features/spectrum-web-components/src/icons-ui.js:16]: please follow alphabetical order.

File: libs/features/spectrum-web-components/src/icons-workflow.js [hunk 2]
```diff
@@ -2,3 +2,4 @@ import '@spectrum-web-components/icons-workflow/icons/sp-icon-help.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-alert.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-link-out-light.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-magnify.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-down.js';
```
Comment (yesil) [inline libs/features/spectrum-web-components/src/icons-workflow.js:5]: please follow alphabetical order.

File: libs/features/spectrum-web-components/src/icons-ui.js (hunk 1 above)
Comment (draganatrajkovic) [reply libs/features/spectrum-web-components/src/icons-ui.js]: Thank you, i pushed fix

File: libs/features/spectrum-web-components/src/icons-workflow.js (hunk 2 above)
Comment (draganatrajkovic) [reply libs/features/spectrum-web-components/src/icons-workflow.js]: Thank you, this is also resolved

**Review by mokimo — APPROVED** [review]

**Review by yesil — CHANGES_REQUESTED** [review]
@draganatrajkovic can you include package-lock.json please?

Also, as per the doc, can you add a simple test?

**Comment by Ben-Zahler** [comment]
@yesil @mokimo we have an unfortunate situation here: the spectrum web component is not working on mobile yet and it seems to be tricky to get it to work reliably (@draganatrajkovic and @saragajic11 tried to import additional dependencies such as "tray", this works 33% of the time).
Furthermore, the mobile behaviour for the "picker" is different than our [reference implementation](https://business.adobe.com/customer-success-stories.html), a tray slides out at the bottom of the screen in the spectrum web component:
image
Everyone in our team thinks the reference implementation provides the better UX.
So we suggest that we close this PR and implement this as per our reference ourselves. Are you ok with this? Or do you see any benefit in adding the "picker" to milo even if we do not need it?

**Comment by yesil** [comment]
@Ben-Zahler before a final decision, I would reach out to SWC team(they are very helpful) at `#spectrum-web-components` and ask their feedback on the topic.
if you need support on the Milo SWC version, let me know, I can try to help there.

**Comment by draganatrajkovic** [comment]
@yesil @mokimo 
Thank you. 
After sync up with our designer we decided not to use picker from spectrum web components.
Therefore I will close this pr.

**Comment by Ben-Zahler** [comment]
@yesil FYI: spectrum web components team did respond quickly in Slack, but the answer is that this is a design request and our designer would have to raise it with the spectrum design team -> after all, the process just seemed to be too complicated for the relatively simple thing we need to implement ourselves.

**Pull request closed without merging**